generator/uscore/uscore_generator.rb

```diff
@@ -131,6 +132,29 @@ def create_authorization_test(sequence)
         sequence[:tests] << authorization_test
       end
 
+      def create_include_test(sequence)
+        include_test = {
+          tests_that: "A Server SHOULD be capable of supporting the following _includes: #{sequence[:include_params].join(', ')}",
+          index: sequence[:tests].length + 1,
+          link: 'https://www.hl7.org/fhir/search.html#include'
+        }
+        first_search = find_first_search(sequence)
+        search_params = first_search.nil? ? 'search_params = {}' : get_search_params(first_search[:names], sequence)
+        include_test[:test_code] = search_params
+        sequence[:include_params].each do |include|
+          included_resource = include.split(':').last.capitalize # kind of a hack, but works for now - would have to otherwise figure out resource type of target profile
+          include_test[:test_code] += %(
+                search_params['_include'] = '#{include}'
+                reply = get_resource_by_params(versioned_resource_class('#{sequence[:resource]}'), search_params)
+                assert_response_ok(reply)
+                assert_bundle_response(reply)
+                #{included_resource.downcase}_results = reply&.resource&.entry&.map(&:resource)&.select { |resource| resource.resourceType == '#{included_resource}' }
```

Comment:
```suggestion
                #{included_resource.downcase}_results = reply&.resource&.entry&.map(&:resource)&.any? { |resource| resource.resourceType == '#{included_resource}' }
```
Then you can just `assert #{included_resource.downcase}_results`